### DISCUSSION TITLE
使用VIP模式，每天早上8点十分的时候IP无法访问，几分钟后恢复正常，是什么原因，如何排查？

### DISCUSSION THREAD

**Discussion by qinjianyun** [openelb/openelb#417] [Q&A]
使用VIP模式，发现每天早上8点十分的时候IP无法访问，几分钟后恢复正常，是什么原因，如何排查？

**Comment by renyunkang** [comment]
看下 openelb-keepalived-vip 中有没有异常日志呢

**Reply by qinjianyun** [reply]
IPVS cmd IP_VS_SO_SET_ADDDEST(1159) erro:Destination already exists 只有这个报错
